pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -49,9 +49,9 @@
     .thegap{style: "height: 1px"}
       &nbsp;
     #desktop-header.desktop-feature
-      = view :header, variant: "desktop"
-    #mobile-header.mobile-feature
-      = view :header, variant: "mobile"
```

Comment:
I looked at this with @Erin007 last week and we couldn't see a need for this any more.  Must have been needed earlier but apparently not any more.